### PR TITLE
Image Puller reorganization + image list

### PR DESCRIPTION
**Pull request by themr0c**
Image puller: 
* Reorganize the chapter to make room for a procedure explaining how to define the list of images to pull.
* Add a procedure explaining how to define the list of images to pull.
* Remove list of images and CLI output that need maintenance.
* Downstream friendly examples, only where necessary.

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [x] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [x] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [x] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/administration-guide/partials/proc_defining-the-list-of-images-to-pull.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[id="defining-the-list-of-images-to-pull_{context}"]
+= Defining the list of images to pull
+
+
+
+.Procedure
+
+include::example${project-context}-prepulled-images-table.adoc[]
```
Comment (tomgeorge) [inline modules/administration-guide/partials/proc_defining-the-list-of-images-to-pull.adoc]: Below, maybe something like

"The first three images are common to all workspaces, and should be included to be pre-pulled. The rest are language-specific plugins. Depending on the language your organization uses, you should add them to the list of images to pre-pull."

File: modules/administration-guide/partials/proc_defining-the-list-of-images-to-pull.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  yq -r '.spec.relatedImages[]'
+----
+====
+endif::[]
++
+.Getting the list of all images for OpenShift
+====
+[subs="+attributes,+quotes,+macros"]
+----
+$ curl -sSLo- https://raw.githubusercontent.com/eclipse/che-operator/master/olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/{prod-ver-patch}/eclipse-che-preview-openshift.v{prod-ver-patch}.clusterserviceversion.yaml | \
```
Comment (themr0c) [inline modules/administration-guide/partials/proc_defining-the-list-of-images-to-pull.adoc]: @tomgeorge @nickboldt Now I need to know how much of this is compatible with downstream.

Comment (tomgeorge) [reply modules/administration-guide/partials/proc_defining-the-list-of-images-to-pull.adoc]: It looks like https://github.com/redhat-developer/codeready-workspaces-operator/blob/crw-2.6-rhel-8/deploy/olm-catalog/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml still has the upstream versions of images, so they must be swapped out in the downstreaming process somewhere.

**Comment by themr0c** [comment]
@l0rd , @nickboldt , @tomgeorge The PR is ready for review.

**Review by tomgeorge — APPROVED** [review]

**Review by yhontyk — APPROVED** [review]

**Pull request merged**